### PR TITLE
[ENHANCE] Update backbone lists

### PR DESCRIPTION
**Pull request by harimkang**
## Summary
Updated the list of replaceable backbones for the latest OTX.

Updated Lib: version (number of backbone)
mmcls: 0.25.0 (18 -> 32)
mmdet: 2.28.1 (12 -> 19)
mmseg: 0.30.0 (20 -> 24)
torchvision: 0.14.1+cu116 (35 -> 56)

'+ Filtering in otx find

```
(otx) otx find --backbone mmdet
+-------+----------------------------------+-------------------+-------------------------------+
| Index |          Backbone Type           |   Required-Args   |            Options            |
+-------+----------------------------------+-------------------+-------------------------------+
|   1   |         mmdet.CSPDarknet         |        arch       |             P5, P6            |
|   2   |          mmdet.Darknet           |                   |                               |
|   3   |      mmdet.DetectoRS_ResNet      |       depth       |          50, 101, 152         |
|   4   |     mmdet.DetectoRS_ResNeXt      |       depth       |          50, 101, 152         |
|   5   |        mmdet.EfficientNet        |        arch       |                               |
|   6   |        mmdet.HourglassNet        |                   |                               |
|   7   |           mmdet.HRNet            |       extra       |                               |
|   8   |        mmdet.MobileNetV2         |                   |                               |
|   9   |  mmdet.PyramidVisionTransformer  |                   |                               |
|   10  | mmdet.PyramidVisionTransformerV2 |                   |                               |
|   11  |           mmdet.RegNet           |        arch       | regnetx_400mf, regnetx_800mf, |
|       |                                  |                   | regnetx_1.6gf, regnetx_3.2gf, |
|       |                                  |                   | regnetx_4.0gf, regnetx_6.4gf, |
|       |                                  |                   |  regnetx_8.0gf, regnetx_12gf  |
|   12  |          mmdet.Res2Net           |       depth       |          50, 101, 152         |
|   13  |          mmdet.ResNeSt           |       depth       |       50, 101, 152, 200       |
|   14  |           mmdet.ResNet           |       depth       |      18, 34, 50, 101, 152     |
|   15  |         mmdet.ResNetV1d          |       depth       |      18, 34, 50, 101, 152     |
|   16  |          mmdet.ResNeXt           |       depth       |          50, 101, 152         |
|   17  |           mmdet.SSDVGG           |     input_size    |            300, 512           |
|       |                                  |       depth       |           11, 16, 19          |
|   18  |      mmdet.SwinTransformer       |                   |                               |
|   19  |       mmdet.TridentResNet        |       depth       |      18, 34, 50, 101, 152     |
|       |                                  |     num_branch    |                               |
|       |                                  |  test_branch_idx  |                               |
|       |                                  | trident_dilations |                               |
+-------+----------------------------------+-------------------+-------------------------------+

(otx) otx find --backbone mmdet --task instance_segmentation
+-------+-------------------------+-------------------+-------------------------------+
| Index |      Backbone Type      |   Required-Args   |            Options            |
+-------+-------------------------+-------------------+-------------------------------+
|   1   |     mmdet.CSPDarknet    |        arch       |             P5, P6            |
|   2   |      mmdet.Darknet      |                   |                               |
|   3   |  mmdet.DetectoRS_ResNet |       depth       |          50, 101, 152         |
|   4   | mmdet.DetectoRS_ResNeXt |       depth       |          50, 101, 152         |
|   5   |    mmdet.HourglassNet   |                   |                               |
|   6   |       mmdet.HRNet       |       extra       |                               |
|   7   |    mmdet.MobileNetV2    |                   |                               |
|   8   |       mmdet.RegNet      |        arch       | regnetx_400mf, regnetx_800mf, |
|       |                         |                   | regnetx_1.6gf, regnetx_3.2gf, |
|       |                         |                   | regnetx_4.0gf, regnetx_6.4gf, |
|       |                         |                   |  regnetx_8.0gf, regnetx_12gf  |
|   9   |      mmdet.Res2Net      |       depth       |          50, 101, 152         |
|   10  |      mmdet.ResNeSt      |       depth       |       50, 101, 152, 200       |
|   11  |       mmdet.ResNet      |       depth       |      18, 34, 50, 101, 152     |
|   12  |     mmdet.ResNetV1d     |       depth       |      18, 34, 50, 101, 152     |
|   13  |      mmdet.ResNeXt      |       depth       |          50, 101, 152         |
|   14  |       mmdet.SSDVGG      |     input_size    |            300, 512           |
|       |                         |       depth       |           11, 16, 19          |
|   15  |   mmdet.TridentResNet   |       depth       |      18, 34, 50, 101, 152     |
|       |                         |     num_branch    |                               |
|       |                         |  test_branch_idx  |                               |
|       |                         | trident_dilations |                               |
+-------+-------------------------+-------------------+-------------------------------+
```

### CODE REVIEW

**Comment by eunwoosh** [comment]
> ## ToDo
> The newly added backbone needs to be verified. I will update the list once more after verification.

Does it mean that this PR will include additional update?

**Comment by harimkang** [comment]
> 

It's not this PR, there will be a new PR later.

File: otx/cli/builder/supported_backbone/mmcls.json [hunk 1]
```diff
@@ -1,5 +1,154 @@
...
+    },
+    "mmcls.DistilledVisionTransformer": {
+      "required": [],
+      "options": {},
+      "available": []
+    },
+    "mmcls.DenseNet": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["121", "169", "201", "161"]
```
Comment (jaegukhyun) [inline otx/cli/builder/supported_backbone/mmcls.json:59]: Is this right order?

File: otx/cli/builder/supported_backbone/mmcls.json [hunk 2]
```diff
@@ -1,5 +1,154 @@
...
+          "large-gf",
+          "large-gf384"
+        ]
+      },
+      "available": []
+    },
+    "mmcls.HRNet": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["w32", "w18", "w30", "w40", "w44", "w48", "w64"]
```
Comment (jaegukhyun) [inline otx/cli/builder/supported_backbone/mmcls.json:111]: I'm not familiar with HRNet, is this right order?

File: otx/cli/builder/supported_backbone/mmcls.json [hunk 3]
```diff
@@ -1,5 +1,154 @@
...
+    },
+    "mmcls.LeNet5": {
+      "required": [],
+      "options": {},
+      "available": []
+    },
+    "mmcls.MlpMixer": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["base", "small", "large"]
```
Comment (jaegukhyun) [inline otx/cli/builder/supported_backbone/mmcls.json:123]: I think "base" would be larger than "small"

File: otx/cli/builder/supported_backbone/mmcls.json (hunk 1 above)
Comment (harimkang) [reply otx/cli/builder/supported_backbone/mmcls.json:59]: I think the order is not important, but I set the first value as default. (This value is also used as default in official mmX.)

File: otx/cli/builder/supported_backbone/mmcls.json [hunk 4]
```diff
@@ -1,5 +1,154 @@
...
+      "required": ["arch"],
+      "options": {
+        "arch": ["small", "large"]
+      },
+      "available": []
+    },
+    "mmcls.MViT": {
+      "required": ["arch"],
+      "options": {
+        "arch": ["base", "tiny", "small", "large"]
```
Comment (jaegukhyun) [inline otx/cli/builder/supported_backbone/mmcls.json:142]: Please check this order

File: otx/cli/builder/supported_backbone/mmcls.json (hunk 2 above)
Comment (harimkang) [reply otx/cli/builder/supported_backbone/mmcls.json:111]: This is the official option order, the first value is the default value. I double checked.

File: otx/cli/builder/supported_backbone/mmcls.json (hunk 3 above)
Comment (harimkang) [reply otx/cli/builder/supported_backbone/mmcls.json:123]: base is default value of arch

File: otx/cli/builder/supported_backbone/mmcls.json (hunk 4 above)
Comment (harimkang) [reply otx/cli/builder/supported_backbone/mmcls.json:142]: base is default value of arch, So I put it like this.

**Review by jaegukhyun — DISMISSED** [review]

**Review by eunwoosh — DISMISSED** [review]

**Comment by harimkang** [comment]
- Classification
```
+-------+----------------------------------+-------+-------+
| Index |          Backbone Type           | build | train |
+-------+----------------------------------+-------+-------+
|   1   |          mmcls.AlexNet           |   O   |   O   |
|   2   |         mmcls.Conformer          |   O   |   X   |
|   3   |         mmcls.ConvMixer          |   O   |   O   |
|   4   |          mmcls.ConvNeXt          |   O   |   X   |
|   5   |         mmcls.CSPDarkNet         |   O   |   X   |
|   6   |         mmcls.CSPResNet          |   O   |   X   |
|   7   |         mmcls.CSPResNeXt         |   O   |   X   |
|   8   | mmcls.DistilledVisionTransformer |   O   |   X   |
|   9   |          mmcls.DenseNet          |   O   |   O   |
|   10  |      mmcls.EfficientFormer       |   O   |   X   |
|   11  |        mmcls.EfficientNet        |   O   |   O   |
|   12  |           mmcls.HorNet           |   O   |   X   |
|   13  |           mmcls.HRNet            |   O   |   O   |
|   14  |           mmcls.LeNet5           |   O   |   X   |
|   15  |          mmcls.MlpMixer          |   O   |   X   |
|   16  |        mmcls.MobileNetV2         |   O   |   O   |
|   17  |        mmcls.MobileNetV3         |   O   |   O   |
|   18  |            mmcls.MViT            |   O   |   O   |
|   19  |         mmcls.PoolFormer         |   O   |   O   |
|   20  |           mmcls.RegNet           |   O   |   O   |
|   21  |         mmcls.RepMLPNet          |   O   |   O   |
|   22  |           mmcls.RepVGG           |   O   |   O   |
|   23  |          mmcls.Res2Net           |   O   |   O   |
|   24  |          mmcls.ResNeSt           |   O   |   O   |
|   25  |           mmcls.ResNet           |   O   |   O   |
|   26  |         mmcls.ResNetV1c          |   O   |   O   |
|   27  |         mmcls.ResNetV1d          |   O   |   O   |
|   28  |        mmcls.ResNet_CIFAR        |   O   |   O   |
|   29  |          mmcls.ResNeXt           |   O   |   O   |
|   30  |          mmcls.SEResNet          |   O   |   O   |
|   31  |         mmcls.SEResNeXt          |   O   |   O   |
|   32  |        mmcls.ShuffleNetV1        |   O   |   O   |
|   33  |        mmcls.ShuffleNetV2        |   O   |   O   |
|   34  |      mmcls.SwinTransformer       |   O   |   O   |
|   35  |     mmcls.SwinTransformerV2      |   O   |   X   |
|   36  |          mmcls.T2T_ViT           |   O   |   X   |
|   37  |        mmcls.TIMMBackbone        |   O   |   X   |
|   38  |            mmcls.TNT             |   O   |   X   |
|   39  |           mmcls.PCPVT            |   O   |   O   |
|   40  |            mmcls.SVT             |   O   |   O   |
|   41  |            mmcls.VAN             |   O   |   O   |
|   42  |            mmcls.VGG             |   O   |   O   |
|   43  |     mmcls.VisionTransformer      |   O   |   X   |
|   44  |         mmdet.CSPDarknet         |   O   |   O   |
|   45  |          mmdet.Darknet           |   O   |   O   |
|   46  |      mmdet.DetectoRS_ResNet      |   O   |   O   |
|   47  |     mmdet.DetectoRS_ResNeXt      |   O   |   O   |
|   48  |        mmdet.EfficientNet        |   O   |   O   |
|   49  |        mmdet.HourglassNet        |   O   |   O   |
|   50  |           mmdet.HRNet            |   O   |   O   |
|   51  |        mmdet.MobileNetV2         |   O   |   O   |
|   52  |  mmdet.PyramidVisionTransformer  |   O   |   O   |
|   53  | mmdet.PyramidVisionTransformerV2 |   O   |   O   |
|   54  |           mmdet.RegNet           |   O   |   O   |
|   55  |          mmdet.Res2Net           |   O   |   O   |
|   56  |          mmdet.ResNeSt           |   O   |   O   |
|   57  |           mmdet.ResNet           |   O   |   O   |
|   58  |         mmdet.ResNetV1d          |   O   |   O   |
|   59  |          mmdet.ResNeXt           |   O   |   X   |
|   60  |           mmdet.SSDVGG           |   O   |   O   |
|   61  |      mmdet.SwinTransformer       |   O   |   O   |
|   62  |       mmdet.TridentResNet        |   O   |   O   |
|   63  |            mmseg.BEiT            |   O   |   O   |
|   64  |         mmseg.BiSeNetV1          |   O   |   O   |
|   65  |         mmseg.BiSeNetV2          |   O   |   O   |
|   66  |           mmseg.CGNet            |   O   |   O   |
|   67  |           mmseg.ERFNet           |   O   |   O   |
|   68  |          mmseg.FastSCNN          |   O   |   O   |
|   69  |           mmseg.HRNet            |   O   |   O   |
|   70  |           mmseg.ICNet            |   O   |   O   |
|   71  |            mmseg.MAE             |   O   |   O   |
|   72  |    mmseg.MixVisionTransformer    |   O   |   O   |
|   73  |        mmseg.MobileNetV2         |   O   |   O   |
|   74  |        mmseg.MobileNetV3         |   O   |   O   |
|   75  |          mmseg.ResNeSt           |   O   |   O   |
|   76  |           mmseg.ResNet           |   O   |   O   |
|   77  |         mmseg.ResNetV1c          |   O   |   O   |
|   78  |         mmseg.ResNetV1d          |   O   |   O   |
|   79  |          mmseg.ResNeXt           |   O   |   O   |
|   80  |     mmseg.STDCContextPathNet     |   O   |   O   |
|   81  |          mmseg.STDCNet           |   O   |   O   |
|   82  |      mmseg.SwinTransformer       |   O   |   O   |
|   83  |        mmseg.TIMMBackbone        |   O   |   O   |
|   84  |           mmseg.PCPVT            |   O   |   O   |
|   85  |            mmseg.SVT             |   O   |   O   |
|   86  |            mmseg.UNet            |   O   |   O   |
|   87  |     mmseg.VisionTransformer      |   O   |   O   |
|   88  |       torchvision.alexnet        |   O   |   O   |
|   89  |    torchvision.convnext_tiny     |   O   |   O   |
|   90  |    torchvision.convnext_small    |   O   |   O   |
|   91  |    torchvision.convnext_large    |   O   |   O   |
|   92  |     torchvision.densenet121      |   O   |   O   |
|   93  |     torchvision.densenet161      |   O   |   O   |
|   94  |     torchvision.densenet169      |   O   |   O   |
|   95  |     torchvision.densenet201      |   O   |   O   |
|   96  |   torchvision.efficientnet_b0    |   O   |   O   |
|   97  |   torchvision.efficientnet_b1    |   O   |   O   |
|   98  |   torchvision.efficientnet_b2    |   O   |   O   |
|   99  |   torchvision.efficientnet_b3    |   O   |   O   |
|  100  |   torchvision.efficientnet_b4    |   O   |   O   |
|  101  |   torchvision.efficientnet_b5    |   O   |   O   |
|  102  |   torchvision.efficientnet_b6    |   O   |   O   |
|  103  |   torchvision.efficientnet_b7    |   O   |   O   |
|  104  |  torchvision.efficientnet_v2_s   |   O   |   O   |
|  105  |  torchvision.efficientnet_v2_m   |   O   |   O   |
|  106  |  torchvision.efficientnet_v2_l   |   O   |   O   |
|  107  |      torchvision.googlenet       |   X   |   X   |
|  108  |     torchvision.inception_v3     |   X   |   X   |
|  109  |      torchvision.mnasnet0_5      |   O   |   O   |
|  110  |     torchvision.mnasnet0_75      |   O   |   O   |
|  111  |      torchvision.mnasnet1_0      |   O   |   O   |
|  112  |      torchvision.mnasnet1_3      |   O   |   O   |
|  113  |     torchvision.mobilenet_v2     |   O   |   O   |
|  114  |  torchvision.mobilenet_v3_large  |   O   |   O   |
|  115  |  torchvision.mobilenet_v3_small  |   O   |   O   |
|  116  |    torchvision.regnet_y_400mf    |   X   |   X   |
|  117  |    torchvision.regnet_y_800mf    |   X   |   X   |
|  118  |    torchvision.regnet_y_1_6gf    |   X   |   X   |
|  119  |    torchvision.regnet_y_3_2gf    |   X   |   X   |
|  120  |     torchvision.regnet_y_8gf     |   X   |   X   |
|  121  |    torchvision.regnet_y_16gf     |   X   |   X   |
|  122  |    torchvision.regnet_y_32gf     |   X   |   X   |
|  123  |    torchvision.regnet_y_128gf    |   X   |   X   |
|  124  |    torchvision.regnet_x_400mf    |   X   |   X   |
|  125  |    torchvision.regnet_x_800mf    |   X   |   X   |
|  126  |    torchvision.regnet_x_1_6gf    |   X   |   X   |
|  127  |    torchvision.regnet_x_3_2gf    |   X   |   X   |
|  128  |     torchvision.regnet_x_8gf     |   X   |   X   |
|  129  |    torchvision.regnet_x_16gf     |   X   |   X   |
|  130  |    torchvision.regnet_x_32gf     |   X   |   X   |
|  131  |       torchvision.resnet18       |   O   |   O   |
|  132  |       torchvision.resnet34       |   O   |   O   |
|  133  |       torchvision.resnet50       |   O   |   O   |
|  134  |      torchvision.resnet101       |   O   |   O   |
|  135  |      torchvision.resnet152       |   O   |   O   |
|  136  |   torchvision.resnext50_32x4d    |   O   |   O   |
|  137  |   torchvision.resnext101_32x8d   |   O   |   O   |
|  138  |   torchvision.resnext101_64x4d   |   O   |   O   |
|  139  |   torchvision.wide_resnet50_2    |   O   |   O   |
|  140  |   torchvision.wide_resnet101_2   |   O   |   O   |
|  141  |  torchvision.shufflenet_v2_x0_5  |   O   |   O   |
|  142  |  torchvision.shufflenet_v2_x1_0  |   O   |   O   |
|  143  |  torchvision.shufflenet_v2_x1_5  |   O   |   O   |
|  144  |  torchvision.shufflenet_v2_x2_0  |   O   |   O   |
|  145  |    torchvision.squeezenet1_0     |   O   |   O   |
|  146  |    torchvision.squeezenet1_1     |   O   |   O   |
|  147  |        torchvision.vgg11         |   O   |   O   |
|  148  |       torchvision.vgg11_bn       |   O   |   O   |
|  149  |        torchvision.vgg13         |   O   |   O   |
|  150  |       torchvision.vgg13_bn       |   O   |   O   |
|  151  |        torchvision.vgg16         |   O   |   O   |
|  152  |       torchvision.vgg16_bn       |   O   |   O   |
|  153  |        torchvision.vgg19         |   O   |   O   |
|  154  |       torchvision.vgg19_bn       |   O   |   O   |
|  155  |       torchvision.vit_b_16       |   X   |   X   |
|  156  |       torchvision.vit_b_32       |   X   |   X   |
|  157  |       torchvision.vit_l_16       |   X   |   X   |
|  158  |       torchvision.vit_l_32       |   X   |   X   |
|  159  |       torchvision.vit_h_14       |   X   |   X   |
|  160  |        torchvision.swin_t        |   O   |   O   |
|  161  |        torchvision.swin_s        |   O   |   O   |
|  162  |        torchvision.swin_b        |   O   |   O   |
|  163  |      torchvision.swin_v2_t       |   O   |   O   |
|  164  |      torchvision.swin_v2_s       |   O   |   O   |
|  165  |      torchvision.swin_v2_b       |   O   |   O   |
|  166  |       torchvision.maxvit_t       |   X   |   X   |
+-------+----------------------------------+-------+-------+
```

**Review by goodsong81 — CHANGES_REQUESTED** [review]
You may want to rebase. The hanging issue was fixed (or avoided) from the 1.0 merge-back.

**Comment by harimkang** [comment]
> You may want to rebase. The hanging issue was fixed (or avoided) from the 1.0 merge-back.

Thank you for letting me know. It's corrected.

**Review by goodsong81 — APPROVED** [review]

**Review by sungmanc — APPROVED** [review]

**Pull request merged**